pkg/kubelet/apis/config/types.go

```diff
@@ -426,6 +426,10 @@ type KubeletConfiguration struct {
 	// +featureGate=MemoryQoS
 	// +optional
 	MemoryThrottlingFactor *float64
+	// CRIVersion for the remote runtime and image endpoint to be used. Can be
+	// either empty (chosen automatically on connection) or set to `v1` and
+	// `v1alpha2` (deprecated).
+	CRIVersion string
```

Comment:
I am not sure why we want to allow configuring this. We can try v1 and fallback to v1alpha1 automatically on connection.

Reply:
@ehashman are we fine not making it configurable?

Reply:
perhaps to simplify testing of v1alpha1?

Reply:
ok, taking it back. We can always test with older containerd version. Adding this flag for one or two releases seems unnecessary